File: PerfTester-Xamarin/PerfTester-Xamarin/Screens/TestList/Cells/TestListCell.cs
using System;

using Foundation;
using PerfTesterXamarin.Models;
using UIKit;

namespace PerfTesterXamarin.Screens.TestList.Cells
{
    public partial class TestListCell : UITableViewCell
    {
        public static readonly NSString Key = new NSString("TestListCell");
        public static readonly UINib Nib;

        static TestListCell()
        {
            Nib = UINib.FromName("TestListCell", NSBundle.MainBundle);
        }

        protected TestListCell(IntPtr handle) : base(handle)
        {
            // Note: this .ctor should not contain any initialization logic.
        }

        public void SetupCellWithTest(Test test)
        {
            TitleLabel.Text = test.Title;
            DescLabel.Text = test.Desc;
            TestImage.Image = UIImage.FromBundle(test.ImageName);
        }
    }
}

File: PerfTester-Xcode/PerfTester-Xcode/Screens/TestList/Cells/TestListCell.swift
//
//  TestListCell.swift
//  PerfTester-Xcode
//
//  Created by Patryk Średziński on 08.06.2018.
//  Copyright © 2018 private.perftesterxcode. All rights reserved.
//

import UIKit

class TestListCell: UITableViewCell {

    @IBOutlet private var testImage: UIImageView!
    @IBOutlet private var titleLabel: UILabel!
    @IBOutlet private var descLabel: UILabel!
    
    func setupCellWithTest(_ test: Test) {
        titleLabel.text = test.title;
        descLabel.text = test.desc;
        testImage.image = UIImage(named: test.imageName);
    }
}
